src/main/webapp/lecture-list.html

```diff
@@ -1,18 +1,85 @@
 <!DOCTYPE html>
 <meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+  integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+  crossorigin="anonymous"
+/>
 <script src="lectures.js" defer></script>
-<form name="lectureForm" action="/lecture" method="post">
-  <h2>Add a new lecture!</h2>
-  <h3>Enter a name for this lecture:</h3>
-  <input type="text" placeholder="Lecture Name" name="name-input" required />
-  <br />
-  <h3>Enter the YouTube link for this lecture:</h3>
-  <input type="url" placeholder="Lecture URL" name="link-input" required />
-  <br />
-  <br />
-  <button type="submit">Submit</button>
-</form>
-<ul id="lecture-list"></ul>
+
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+    <button type="button" class="btn btn-outline-primary">Logout</button>
+  </nav>
+
+  <div id="list-cont">
+    <div id="text-cont">
+      <h2 class="text-center">Active Lectures</h2>
+      <p class="text-center">
+        Select a lecture below or add a lecture to start watching and interacting with other students!
+      </p>
+    </div>
+    <div id="lecture-list" class="list-group"></div>
+  </div>
+
+  <div id="add-btn-cont">
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addLectureModal">
+      Add a Lecture
+    </button>
+  </div>
+</body>
+
+<footer id="footer">
+  <div class="container">
+    <p class="text-center">
+      <a href="#" class="link">About</a> | <a href="#" class="link">Feedback</a> |
```

Comment:
Can you make About redirect to "/"?  And Feedback redirect to our github issues page?

Reply:
added